app/components/Chat.vue.ts

```diff
@@ -71,6 +71,30 @@ export default class Chat extends Vue {
         `, true);
       }
     });
+    webview.addEventListener('new-window', e => {
+      const protocol = url.parse(e.url).protocol;
+
+      if (protocol === 'http:' || protocol === 'https:') {
+        electron.remote.shell.openExternal(e.url);
+      }
+    });
+
+    webview.addEventListener('dom-ready', () => {
+      webview.setZoomFactor(settings.chatZoomFactor);
+
+      if (settings.enableFFZEmotes && this.isTwitch) {
+        webview.executeJavaScript(`
+          
+          localStorage.setItem('ffz_clickTwitchEmotes', true);
+          localStorage.setItem('ffz_darkenedMode', ${ settings.nightMode ? 'true' : 'false' });
+          
+          var ffzscript1 = document.createElement('script');
+          ffzscript1.setAttribute('src','https://cdn.frankerfacez.com/script/script.min.js');
+          document.head.appendChild(ffzscript1);
+        `, true);
+      }
+    });
```

Comment:
We already have `dom-ready` listener above. Don't create listeners when you can reuse a created one. Your code should looks like:
```
if (this.isTwitch) {

  if (settings.enableBTTVEmotes) {
    // do stuff with BBTV....
  }

  if (settings.enableFFZEmotes) {
        // do stuff with FFZ....
  }


}


```
Did you check that both plugins work well? If they dont, you need to replace checkboxes in the settings to listbox or radiobuttons